pom.xml

```diff
@@ -18,7 +18,7 @@
         <attach.version>4.0.11</attach.version>
         <connect.version>2.0.1</connect.version>
         <javafx.maven.plugin.version>0.0.6</javafx.maven.plugin.version>
-        <gluonfx.maven.plugin.version>1.0.0</gluonfx.maven.plugin.version>
+        <gluonfx.maven.plugin.version>1.0.1-SNAPSHOT</gluonfx.maven.plugin.version>
```

Comment:
This change is not required (it won't work unless you add the pluginRepository).

Reply:
The plugin repository is already present in the [parent pom](https://github.com/gluonhq/gluon-samples/blob/master/pom.xml#L53)